src/StereoDB/StereoDb.fs

```diff
@@ -122,10 +135,53 @@ type internal StereoDb<'TSchema when 'TSchema :> IDbSchema>(logger: ILogger, sch
             with
                 ex -> ()
     }
-    
+
+    let createHeartbeatTopic (config: HeartbeatConfig) = valueTask {
+        use adminClient = AdminClientBuilder(
+            AdminClientConfig(BootstrapServers = config.KafkaBootstrapServers)).Build()
+
+        try
+            // let! existingTopics = adminClient.DescribeTopicsAsync(TopicCollection.OfTopicNames([_heartbeatTopic]))
+            // let heartbeatTopicExists = existingTopics.TopicDescriptions.Exists(
+            //     fun t -> t.Name = _heartbeatTopic && not t.Error.IsError)
+            // if heartbeatTopicExists then
+            //     return! ValueTask.FromResult()
+            let! r = adminClient.CreateTopicsAsync(
```

Comment:
you can use
```let _ = adminClient.CreateTopicsAsync```
I suppose you don't even need 
```r |> ignore```

Reply:
rewritten with `do! adminClient.CreateTopicsAsync`